applications/tari_indexer/src/json_rpc/handlers.rs

```diff
@@ -540,6 +547,47 @@ impl JsonRpcHandlers {
         }
     }
 
+    pub async fn get_epoch_manager_stats(&self, value: JsonRpcExtractor) -> JrpcResult {
+        let answer_id = value.get_answer_id();
+        let current_epoch = self.epoch_manager.current_epoch().await.map_err(|e| {
+            JsonRpcResponse::error(
+                answer_id,
+                JsonRpcError::new(
+                    JsonRpcErrorReason::InvalidParams,
```

Comment:
InvalidParams is used to indicate that the user passed in something invalid. JsonRpcErrorReason::InternalError is  more appropriate. You can use the `Self::internal_error` helper. Same for the others below

Reply:
It was copy-pasted, so I changed the original as well.